netboot-oci-artifacts.md

```diff
@@ -0,0 +1,96 @@
+# Packaging Network Boot files
+
+Netboot files (i.e. PXE) can be packaged as [OCI Artifacts](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md)
+using an [OCI image manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md)
+and stored in OCI registry.
+
+## Image config object
+
+The configuration object must have the MIME type `application/vnd.oci.empty.v1+json`, and must be
+an empty JSON object ('{}').
+See the [Guidance for an empty descriptor](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidance-for-an-empty-descriptor) 
+
+Because the `config.mediaType` is set to the empty value, the `artifactType` MUST be defined and
+currently it has MIME type `application/vnd.unknown.artifact.v1`.
```

Comment:
Optionally, we could come up with a specific mime type but then we'd need register it with https://www.iana.org/assignments/media-types/media-types.xhtml. For simplicity I thought we can for now stick to the default one.